_projects/apb.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: page
+title: APB VIP
+description: VIP for APB protocol.
+img: assets/img/apb.png
+importance: 1
+contributors: Muneeb
+redirect: https://github.com/mbits-mirafra/apb_avip
+category: work
+---
+
+Every project has a beautiful feature showcase page.
```

Comment:
You can remove all the stuff from line 11 onwards as we are not using individual project pages.

Reply:
Also would be nice to add more contributors and their profiles.